catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/test/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformerTest.java

```diff
@@ -241,7 +241,7 @@ private void assertDefaultAttributesExist(Metacard metacard) {
 
   private void assertExpectedAttributes(Metacard metacard) {
     assertThat(getAttributeValue(Location.COUNTRY_CODE, metacard), equalTo("CC - GBR"));
-    assertThat(getAttributeValue(Core.LOCATION, metacard), equalTo("POINT (-123.26 49.39)"));
+    assertThat(getAttributeValue(Core.LOCATION, metacard), equalTo("POINT (49.39 -123.26)"));
```

Comment:
❓ why is the metacard location expected to be lat/lon ordering?

Reply:
I think @jlcsmith is referring to https://github.com/codice/ddf/blob/ed1a59e34036f2d612f1b77999456a656c397dae/distribution/docs/src/main/resources/content/_metadataReference/core-attributes-table.adoc. DDF expects WKT to be in lon/lat order.

Reply:
well said @brendan-hofmann